File: class01/cpp/AP0111-AreaRetanguloExtra.cpp
#include <iostream>
#include <iomanip>
int main() {
    double base, altura;
    std::cout << "Digite a base: "; std::cin >> base;
    std::cout << "Digite a altura: "; std::cin >> altura;
    double area = base * altura;
    std::cout << "A área do retângulo é " << std::fixed << std::setprecision(2) << area << std::endl;
    return 0;
}

File: class01/java/AP0111-AreaRetanguloExtra.java
import java.util.Scanner;
public class AreaRetanguloExtra {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite a base: ");
        double base = sc.nextDouble();
        System.out.print("Digite a altura: ");
        double altura = sc.nextDouble();
        double area = base * altura;
        System.out.printf("A área do retângulo é %.2f%n", area);
        sc.close();
    }
}